_posts/2021-09-26-boa-release-13.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title: "Boa release v0.13"
+author: Boa Developers
+---
+
+Boa v0.13 is here! Boa is a JavaScript parser, compiler and executor written in the Rust programming language. It makes it easy to embed a JS engine in your projects, and you can even use it from webassembly. See the [About](/about) page for more info.
+
+We currently support part of the language. In this release, our conformance has grown to 41.97% of the official ECMAScript Test Suite (Test262). In this release, we have closed ?? issues and merged ?? pull requests. You can check the full list of changes [here](https://github.com/boa-dev/boa/blob/v0.13/CHANGELOG.md).
```

Comment:
The 0.13 milestone has 40 issues closed and 105 PRs merged + 1 changelog PR. Those should be the final numbers I think.